AlwaysOnKotlin/compose/build.gradle

```diff
@@ -86,6 +86,9 @@ dependencies {
     implementation libs.wear.compose.material
     implementation libs.androidx.window
     implementation libs.compose.ui.test.manifest
+    
+    // Release mode doesn't work without that dependency
+    compileOnly "com.google.android.wearable:wearable:2.9.0"
```

Comment:
Should we make this implementation?

Reply:
that's not possible the implementation is only on the device from what I understood